docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -160,6 +161,8 @@ Within the Gradle Build Server I have added unit tests and integration tests to
   </div>
 </div>
 
+[//]: # (TODO: Attach end term demo video once published)
```

Comment:
Sorry, I still haven't finished it yet. Will do on the weekend

Reply:
No worries. I just left a todo to remind myself later.